packages/tldraw/src/test/frames.test.ts

```diff
@@ -110,6 +116,22 @@ describe('creating frames', () => {
 		})
 	})
 
+	it('parents a shape when drag-creating a frame over it', () => {
+		const rectId: TLShapeId = createRect()
```

Comment:
In general I'm all for pulling out logic, but on this case it makes it a bit harder to understand the test? I need to know the values of the default params to understand the code. Maybe we can always require the arguments so that you don't have to jump to `createRect` (and `dragCreateFrame`) to understand the test.